### DISCUSSION TITLE
Is possible to add text area support

### DISCUSSION THREAD

**Discussion by hawktang** [widgetti/solara#98] [Ideas]
Is possible to add text area support

which allow multiline input instead single line only

**Comment by maartenbreddels** [comment]
Good question. We should add that!
Currently, you can take a look at https://github.com/widgetti/solara/blob/a99283cd0afdd6f0570ee922086425ef3e0a614a/solara/template/markdown.py#L30 for how to do that with vuetify, but I think we want a component in solara as well.

**Comment by maartenbreddels** [comment]
See also https://github.com/widgetti/solara/issues/126
